src/Core/Models/Api/Request/Organizations/OrganizationKeysRequestModel.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Bit.Core.Models.Table;
+using System.ComponentModel.DataAnnotations;
+using Bit.Core.Models.Business;
+
+namespace Bit.Core.Models.Api
+{
+    public class OrganizationKeysRequestModel
+    {
+        public string PublicKey { get; set; }
+        [Required]
```

Comment:
why only the private key required?

Reply:
That's a great question. That's how the `KeysRequestModel` is structured, but I'm not sure if that's by mistake or on purpose.

Reply:
@kspearrin Do you see any issue with me adding a `required` decorator to the `PublicKey` as well?

Reply:
No. You can then avoid alot of those validators, i think

Reply:
Updated!